src/shared/lib/MutationalSignaturesUtils.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import _ from 'lodash';
+import {IMutationalSignature, IMutationalSignatureMeta, ISignificantMutationalSignaturesForSample} from "../model/MutationalSignature";
+
+const MIN_NUMBER_OF_MUTATIONS_THRESHOLD = 5;
+const MIN_NUMBER_OF_MUTATIONS_STATEMENT = "There are not enough mutations to call mutational signatures for this case.";
+const DUBIOUS_NUMBER_OF_MUTATIONS_THRESHOLD = 15;
+const DUBIOUS_NUMBER_OF_MUTATIONS_STATEMENT = "This case has a low number of mutations, signatures are highly uncertain.";
+
+
+export function progressBar(confidence:string, determineColor:boolean) {
+    let confidencePerc = Math.round(parseFloat(confidence) * 100);
+
+    // Coloring same as CVR as discussed with @anoopbr
```

Comment:
the coloring should be ignored, or it should be something specifically for mutational signatures (this coloring was for GDD)

Reply:
The coloring for the bar is set to only gray right now. I can remove the comment to avoid confusion.